testplan/runners/local.py

```diff
@@ -25,24 +25,28 @@ def __init__(self, **options):
     def _execute(self, uid):
         """Execute item implementation."""
         # First retrieve the input from its UID.
-        target = self._input[uid]
+        runnable = self._input[uid]
 
-        # Inspect the input type. Tasks must be materialized before they can be
-        # run.
-        if isinstance(target, tasks.Task):
-            runnable = target.materialize()
-            if not runnable.parent:
-                runnable.parent = self
-            if not runnable.cfg.parent:
-                runnable.cfg.parent = self.cfg
-        elif isinstance(target, entity.Runnable):
-            runnable = target
+        # Inspect the input type. Tasks must be materialized before
+        # they can be run.
+        if isinstance(runnable, entity.Runnable):
+            target = runnable
+        elif isinstance(runnable, tasks.Task):
+            target = runnable.materialize()
```

Comment:
swapping target <==> runnable?

I think it makes better sense to say:
runnable = target.materialize()

Reply:
That makes sense. In LocalRunner target is materialized to a runnable entity, while in TestRunner the runnable is materialized to a target, They have opposite style. So I will change the code in TestRunner, a entity/task/callable to be added is a target, and it can be materialized to a runnable object.